LongReadRNABenchmark/LongReadRNABenchmark.wdl

```diff
@@ -0,0 +1,205 @@
+version 1.0
+
+import "LongReadRNABenchmarkTasks.wdl" as LongReadRNABenchmarkTasks
+
+workflow LongReadRNABenchmark {
```

Comment:
I wonder if it makes sense to split this apart.  There could be a single wdl that takes in gtfs and benchmarks them, as well as other wdls which can run isoform construction for particular tools.  You could also have a wrapper that combines them all together (ie into basically this wdl).

Reply:
and I think it would be good to remove the hardcoding of specific tools as much as possible.  otherwise we are stuck in a pretty static state of only testing these specific tools.

Reply:
Most hardcoding (if not all) that could be avoided has been removed. Marking resolved for now, we could revisit the splitting main WDL into benchmarking and tool-running if desired.